Readme.md

```diff
@@ -42,7 +44,7 @@ make build
 
 To start the public available image run:
 ```bash
-docker run -it -v "$(pwd)/config.yaml:/config.yaml"  -p 8002:8002 hikhvar/mqtt2prometheus:latest 
+docker run -it -v "$(pwd)/config.yaml:/config.yaml"  -p 8002:8002 docker.pkg.github.com/hikhvar/mqtt2prometheus/mqtt2prometheus:latest 
```

Comment:
I think this needs to reference the port 9641 which is the default. as per line 76.

Reply:
Thank you for the review! I changed it.